README.md

```diff
@@ -5,6 +5,7 @@ Extendable Tiny Datacenter Hypervisor on top of Mesos architecture. Wakame-vdc v
 ## Build
 
 Ensure ``$GOPATH`` is set. ``$PATH`` needs to have ``$GOPATH/bin``.
+The developer package of lxc is also required. (lxc-devel from epel-release on redhat systems, or lxc-dev on debian systems) package is also required.
```

Comment:
I think an extra newline between GOPATH and lxc-dev would make it easier to read but not gonna hold back the PR for that.